spec/helpers/administrate/application_helper_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe Administrate::ApplicationHelper do
+  describe "#display_resource_name" do
+    it "defaults to the plural of the model name" do
+      displayed = display_resource_name(:customer)
+
+      expect(displayed).to eq("Customers")
+    end
+
+    it "handles string arguments" do
+      displayed = display_resource_name("customer")
+
+      expect(displayed).to eq("Customers")
+    end
+
+    it "handles plural arguments" do
+      displayed = display_resource_name(:customers)
+
+      expect(displayed).to eq("Customers")
+    end
+
+    context "when translations are defined" do
+      it "uses the plural of the defined translation" do
+        translations = {
+          activerecord: {
+            models: {
+              customer: {
+                one: "User",
+                other: "Users",
+              } } } }
```

Comment:
What do you feel about putting every `}` on its own line? This looks rather Lisp-y.